### DISCUSSION TITLE
GitHub for student and 2FA

### DISCUSSION THREAD

**Discussion by antoninirbe** [community/community#106790] [GitHub Education]
### Hi everyone,

Hello,

We have teachers who would like to use GitHub with students, but we cannot expect all students to have phones for 2FA. What are other teachers for student authentication. Thank you!

**Comment by gil-son** [comment]
Hello,

Is possible use Authenticator on Google Chrome:

1. Open your Google Chrome and access the [Chrome Web Store](https://chromewebstore.google.com/)

2.  Search to Autenticator
![gc1](https://github.com/community/community/assets/72712095/5ee31586-0c25-447c-a41f-5a5bca1206de)

3.  Click on Use in Chrome and Add the extension
![gc2](https://github.com/community/community/assets/72712095/b8504c16-d460-479a-9c41-8350341c977b)

4. In this video you can see [how to configure the Authenticator](https://www.youtube.com/watch?v=x14KazisIAQ)

If I supported you, consider to mark my answer
